src/stability_sdk/animation.py

```diff
@@ -222,7 +222,7 @@ def apply_inpainting(self, mask: np.ndarray, prompts: List[str], weights: List[f
                 steps=inpaint_steps,
                 seed=seed,
                 cfg_scale=self.args.cfg_scale,
-                blur_ks=11,
+                blur_radius=5,
```

Comment:
oh no! I liked the blurks! I just thought it was funny :( oh well, this is way clearer. We'll sneak blurks back into the codebase somewhere